api/app/settings/common.py

```diff
@@ -764,3 +764,5 @@
 # Real time(server sent events) settings
 SSE_SERVER_BASE_URL = env.str("SSE_SERVER_BASE_URL", None)
 SSE_AUTHENTICATION_TOKEN = env.str("SSE_AUTHENTICATION_TOKEN", None)
+
+DISABLE_INVITE_LINKS = env.str("DISABLE_INVITE_LINKS", None)
```

Comment:
Shouldn't this be bool?

Reply:
Yes, yes it should! I've updated that now.